app/src/main/java/org/oppia/app/profile/AdminAuthActivityPresenter.kt

```diff
@@ -61,6 +65,18 @@ class AdminAuthActivityPresenter @Inject constructor(
     }
   }
 
+  fun handleOnSavedInstanceState(bundle: Bundle) {
+    bundle.putString(KEY_ADMIN_AUTH_INPUT_ERROR_MESSAGE, authViewModel.errorMessage.get())
+  }
+
+  fun handleOnRestoreInstanceState(bundle: Bundle) {
+    val errorMessage = bundle.getString(KEY_ADMIN_AUTH_INPUT_ERROR_MESSAGE)
+    Log.d("Debug", "Error message = $errorMessage")
+    if(errorMessage != null && errorMessage.isNotEmpty()) {
```

Comment:
Nit spacing issues: Update to `if (errorMessage != null && errorMessage.isNotEmpty()) {`

space after `if`

Reply:
Done